run/command.go

```diff
@@ -78,7 +78,7 @@ func (r *runner) Get(w io.Writer, subCmdArgs []string) (help.HelpMessage, error)
 		if err != nil {
 			return help.MsgNone, err
 		}
-		r.GoCmd("install", []string{a})
```

Comment:
I think the `a` on this line refers to the argument from range, not an "-a" argument. From what I see it will never pass "-a" into the unless the user passes it in...